### ISSUE TITLE
copr-dist-git fails to build on rawhide

### ISSUE THREAD

**Issue by nikromen** [fedora-copr/copr#2608]
https://download.copr.fedorainfracloud.org/results/@copr/copr-pull-requests:pr:2600/fedora-rawhide-x86_64/05738991-copr-dist-git/builder-live.log.gz

my wild guess is that some dependent package is updated in rawhide with new unexpected behavior and causes trouble. I think it will be some package inside rpkg but it requires further investigation.

**Comment by nikromen** [comment]
ok seems like it is likely new update of rpkg itself https://bodhi.fedoraproject.org/updates/FEDORA-2023-5cec4602e4

**Comment by nikromen** [comment]
update:

rpkg 1.66-5 release introduced more robust spec file checking that our tests don't like. I investigated a bit and I managed to identify the issue. And it seems like the problem is already reported here https://pagure.io/rpkg/issue/684

The new change which breaks our tests: https://pagure.io/rpkg/c/1108810bdefd0d880517b274acd6a3bd0d4156e0?branch=master

The issue is that `self.layout` has `IncompleteLayout` type for our tests.
This is the condition which then raises no spec file error in our tests: https://pagure.io/rpkg/blob/1108810bdefd0d880517b274acd6a3bd0d4156e0/f/pyrpkg/__init__.py#_1168


TODO:
- [ ] investigate whether this is bug on our side (e.g. we have some bug in tests so the spec file is not there) and what could be the workaround
- [ ] unskip the tests from #2652
